src/Http/Features/src/IPersistentStateFeature.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Http.Features
```

Comment:
Should this namespace be `Microsoft.AspNetCore.Connections.Features`?

These feature is kind of used by both HTTP layer and connection layer, but it is common for a connection feature to be used by HTTP, but never for a HTTP feature to be used by connections.

Reply:
I like `Microsoft.AspNetCore.Connections.Features`

Reply:
This is assuming we can move this to Connection.Abstractions.

I don't think this belongs in the Microsoft.Extensions.Features assembly. This probably belongs in Microsoft.AspNetCore.Http.Features or Microsoft.AspNetCore.Connections.Abstractions.

I initially thought this was already in Microsoft.AspNetCore.Http.Features, but that would be in `/srs/Http/Http.Features/` not `/src/Http/Features/`.

@Tratcher What do you think about renaming the folder from `/src/Http/Features/` to `/src/Http/Extensions.Features/`?

Reply:
That folder naming has been confusing. What about `/src/Extensions/Features/`?

Reply:
👍 I prefer that.

Reply:
Changed namespace to `Microsoft.AspNetCore.Connections.Features` and moved to `Connections.Abstractions.

Reply:
Yes please 👍🏾

Reply:
Filed https://github.com/dotnet/aspnetcore/issues/34423 for moving Http/Features to Extensions/Features.